Library/Homebrew/formula.rb

```diff
@@ -1543,7 +1543,8 @@ def std_configure_args
   # Standard parameters for cargo builds.
   sig { params(root: T.any(String, Pathname), path: String).returns(T::Array[T.any(String, Pathname)]) }
   def std_cargo_args(root: prefix, path: ".")
-    ["--locked", "--root", root, "--path", path]
+    ["--locked", "--root", root, "--path", path, "--config",
+     "build.rustflags=[#{Hardware.rustflags_target_cpu}]"]
```

Comment:
How does this interact with `RUSTFLAGS` set in the environment? Does it override them? Append to them?

I think we may prefer to set `RUSTFLAGS` in the environment as well as/instead of doing this, since not all `rustc` invocations happen through a `cargo` call that uses `std_cargo_args`. See, for example, `gstreamer`, or the many formulae that have a `cryptography` resource.